recipes/knit/LICENSE

```diff
@@ -0,0 +1,28 @@
+Copyright (c) 2016, Continuum Analytics, Inc. and contributors
```

Comment:
`license_file` is pulling this from the source code (without issues). So there is no need to include the license file in the recipe.

Reply:
Should I remove it after all?

Reply:
Yes, please remove this file only.